docs/dev/analysis/features/table/table-cell.rst

```diff
@@ -7,6 +7,20 @@ properties affecting its size, appearance, and how the content it contains is
 formatted.
 
 
+Cell.vAlign
+-----------
+
+Candidate protocol::
+
+    >>> from docx.enum.table import WD_VERTICAL_ALIGNMENT
+    >>> row = table.add_row()
+    >>> row.cells[0]
+    <docx.table._Cell object at 0x...>
+    >>> row.cells[0].vAlign
+    None
+    >>> row.cells[0].vAlign = WD_VERTICAL_ALIGNMENT.CENTER
+
```

Comment:
I'll just rename the enum to WD_ALIGN_VERTICAL, I have no preference of one over the other.